antarest/study/storage/variantstudy/model/command/create_binding_constraint.py

```diff
@@ -35,6 +35,12 @@
 }
 
 
+class TermMatrices(str, Enum):
```

Comment:
This is good but I have a lot of warnings in my IDE when calling `TermMatrices.value` because it understands TermMatrices as a string :/ Perhaps try to make this class inherit from Enum only to see if the code still works as it will shush the warnings